lib/ansible/modules/monitoring/zabbix_map.py

```diff
@@ -0,0 +1,800 @@
+        C(zbx_trigger) contains name of the trigger used as a link indicator in C(host_name:trigger_name) format.
+            More than one trigger could be specified by adding some prefix (e.g., C(zbx_trigger1), C(zbx_trigger2)).
+        C(zbx_trigger_color) contains indicator color specified either as CSS3 name or as a hexadecimal code starting with C(#).
+        C(zbx_trigger_draw_style) contains indicator draw style. Possible values are the same as for C(zbx_draw_style)."
+requirements:
+    - "python >= 2.6"
+    - zabbix-api
+    - pydotplus
+    - webcolors
+    - PIL
+    - Graphviz
+version_added: "2.4"
+options:
+    server_url:
+        description:
+            - Url of Zabbix server, with protocol (http or https).
+        required: true
+        aliases: [ "url" ]
+    login_user:
+        description:
+            - Zabbix user name, used to authenticate against the server.
+        required: true
+    login_password:
+        description:
+            - Zabbix user password.
+        required: true
+    http_login_user:
+        description:
+            - Basic Auth login.
+        required: false
+        default: None
+    http_login_password:
+        description:
+            - Basic Auth password.
+        required: false
+        default: None
+    name:
+        description:
+            - Name of the map.
+        required: true
+    data:
+        description:
+            - Graph written in DOT language.
+        required: false
+        aliases: [ "dot_data" ]
+    state:
+        description:
+            - State of the map.
+            - On C(present), it will create if map does not exist or update the map if the associated data is different.
+            - On C(absent) will remove the map if it exists.
+        required: false
+        choices: ['present', 'absent']
+        default: "present"
+    timeout:
+        description:
+            - The timeout of API request (seconds).
+        required: false
+        default: 10
+    width:
+        description:
+            - Width of the map.
+        required: false
+        default: 800
+    height:
+        description:
+            - Height of the map.
+        required: false
+        default: 600
+    margin:
+        description:
+            - Size of white space between map's borders and its elements.
+        required: false
+        default: 40
+    expand_problem:
+        description:
+            - Whether the the problem trigger will be displayed for elements with a single problem.
+        required: false
+        default: true
+    highlight:
+        description:
+            - Whether icon highlighting is enabled.
+        required: false
+        default: true
+    label_type:
+        description:
+            - Map element label type.
+        required: false
+        choices: ['label', 'ip', 'name', 'status', 'nothing', 'custom']
+        default: "name"
+'''
+
+RETURN = ''' # '''
```

Comment:
While return values should be documented for new modules, the other zabbix modules do not document this either (as yet). However, there should not be a need to explicitly define an empty variable.

Reply:
[Documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block) says: 
> If your module doesn’t return anything (apart from the standard returns), this section of your module should read: ```RETURN = ''' # '''```